scripts/build/version_info.js

```diff
@@ -0,0 +1,80 @@
+// GATHER VERSION INFO FROM COMPONENTS
+const path = require('path');
+const fs = require('fs');
+const {cwd} = process;
+
+// helpers
+const clean = name => name.replace(/^@axa\-ch\//, 'axa-');
+
+const justVersionInfo = info => info.replace(/[^A-Za-z0-9\.]/g, ''); // deliberate consequence: ignore caret
+
+const pick = (object, ignore = {}) => {
+  const result = {};
+  const components = Object.keys(object).filter( key => key.indexOf('@axa-ch/') === 0 && !ignore[key]);
+  components.forEach( name => {
+    result[clean(name)] = justVersionInfo(object[name]);
+  });
+  return result;
+}
+
+const ignoreNonComponents = (packages, ignores) => packages.filter( package => {
+    // ignore components classified under our Atomic-Design scheme as starting with 0
+    // (i.e. currently 00-materials, 05-utils)
+    const ignore = package.indexOf('src/components/0') > -1;
+    if (ignore) {
+        const [_, lastPathComponent] = package.match(/([a-z]+)$/);
+        if (!lastPathComponent) {
+            return true;
+        }
+        const dependencyStylePackage = `@axa-ch/${lastPathComponent}`;
+        ignores[dependencyStylePackage] = true;
+    }
+    return !ignore;
+});
+
+const parseLernaJSON = aDirectory => {
+    // read lerna.json in directory passed to get all relevant components
+    const directoryPath = `${aDirectory}${path.sep}`;
+    const lernaFilePath = `${directoryPath}lerna.json`;
+    const lernaJSON = require(lernaFilePath);
+    const {packages} = lernaJSON;
+    // iterate over all component packages in lerna.json,
+    // and partition into components proper and to-be-ignored ones
+    const ignore = {};
+    const components = ignoreNonComponents(packages, ignore);
+    return {ignore, components, directoryPath};
+};
+
+// rollup version, is invoked in each component directory
+const gatherVersions = (aPath = cwd(), versionInfo = {}, ignore) => {
+    // determine to-be-ignored dependencies, if not passed-in already
+    if (ignore === undefined) {
+        ({ignore} = parseLernaJSON(path.resolve(aPath, '..', '..', '..', '..')));
+    }
+    // construct full path to component package.json
+    const filePath = `${aPath}${path.sep}package.json`;
+    // bail out if package.json missing
+    if (!fs.existsSync(filePath)) {
+        return '{}';
```

Comment:
@markus-walther better like this: `return JSON.stringify({})`

Reply:
Done.